src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {AuthLoginInfo} from "./auth-login-info";
+import {SignUpInfo} from "./sign-up-info";
+import {JwtHelperService} from "@auth0/angular-jwt";
+import {TokenStorageService} from "./token-storage.service";
+
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  private loginUrl = 'http://localhost:8080/api/v1/auth/login';
```

Comment:
create a file with constants

Reply:
defined clobal constants in common/AppConfig.ts